package.json

```diff
@@ -25,7 +25,11 @@
     "vector database",
     "semantic search"
   ],
-  "files": ["dist", "README.md", "LICENSE"],
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE"
+  ],
```

Comment:
not sure why `npm run format:fix` changed so much, maybe it just hasn't been run in a while